_data/projects/blockstack-core.yml

```diff
@@ -0,0 +1,13 @@
+name: Blockstack Core
+desc: The Blockstack Core is the reference implementation of Blockstack
+site: https://blockstack.org
+tags:
+- python
+- blockstack
+- blockchain
+- dapp
+- identity
+- DNS
+upforgrabs:
+  name: good first task
+  link: https://github.com/blockstack/blockstack-core/labels/help-wanted
```

Comment:
The link here should be in sync with the `name` value above - not sure which label you want to use here. I can only see one open `help-wanted` issue open, and `good first task` doesn't exist on the label.